snap/snapcraft.yaml

```diff
@@ -334,6 +332,11 @@ apps:
       - network-bind
 
 parts:
+  version:
+    plugin: nil
+    source: .
+    override-pull: |
```

Comment:
also as we discussed, it would be great if you could leave a comment explaining how this prevents pulling all of the source for the git tree from being staged into the part dir.